packages/dtx-web/src/lib/services/fileManager.ts

```
@@ -0,0 +1,51 @@
+	 */
+	static generateKey(simfileId: string | null, fileName: string): string {
+		return `${simfileId || 'local'}:${fileName}`;
+	}
+
+	/**
+	 * Get all stored file keys
+	 */
+	static getKeys(): string[] {
+		return Array.from(this.files.keys());
+	}
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)